.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
 matrix:
   include:
     - os: linux
-      dist: bionic
+      dist: trusty
```

Comment:
I think we could test this on macOS too by moving the env line up top and then adding some extra downloads to the uname == Darwin section below (just download from llvm.org like we do for 3.8 and 3.5; ok to do it unconditionally).

Reply:
Fixed. However I cannot find a prebuilt package in the 6.0.1 tree for macOS, so I just used 6.0.0